.github/workflows/python_wheels.yml

```diff
@@ -0,0 +1,106 @@
+name: Python Wheels
+
+# TODO: Shrink this down to pushes to master, perhaps. During development, this
+# needs to run more often to work out bugs.
+on:
+  - push
+
+jobs:
+  build_linux_wheel:
+    strategy:
+      matrix:
+        # PEP 425 Python version tags - eg 'cp38-cp38' is a CPython 3.8 build.
+        python_version_tag:
+         - cp36-cp36m
+         - cp37-cp37m
+         - cp38-cp38
+         - cp39-cp39
+    runs-on: ubuntu-latest
+    container:
+      image: quay.io/pypa/manylinux_2_24_x86_64
+
+    steps:
+      - name: Install dependencies
+        run: |
+          apt-get update -y
+          apt-get install gfortran liblapack-dev -y
+
+      - name: Select Python Version
+        run: |
+          # Adds the selected Python version to the system path
+          echo "/opt/python/${{ matrix.python_version_tag }}/bin" >> $GITHUB_PATH
+      - name: Install numpy
+        run: |
+          pip install numpy
+
+      - uses: actions/checkout@v2
+
+      - name: Configure openorb
+        run: |
+          ./configure gfortran opt --with-pyoorb --with-f2py=f2py --with-python=python
+
+      - name: Build wheel
+        run: |
+          cd python
+          python setup.py bdist_wheel
+
+      - name: Emit wheel
+        uses: actions/upload-artifact@v2
+        with:
+          name: manylinux-${{ matrix.python_version_tag }}-wheel
+          path: python/dist/*.whl
+  verify_linux_wheel:
+    needs: build_linux_wheel
+    strategy:
+      matrix:
+            # PEP 425 Python version tags - eg 'cp38-cp38' is a CPython 3.8 build.
+        python_version_tag:
+         - cp36-cp36m
+         - cp37-cp37m
+         - cp38-cp38
+         - cp39-cp39
+    runs-on: ubuntu-latest
+    steps:
+
+      # The manylinux builder uses PEP
+      - if: ${{ matrix.python_version_tag == 'cp36-cp36m' }}
+        name: Install Python 3.6
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.6
+
+      - if: ${{ matrix.python_version_tag == 'cp37-cp37m' }}
+        name: Install Python 3.7
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+
+      - if: ${{ matrix.python_version_tag == 'cp38-cp38' }}
+        name: Install Python 3.8
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+
+      - if: ${{ matrix.python_version_tag == 'cp39-cp39' }}
+        name: Install Python 3.9
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
```

Comment:
I'm a little hesitant about these hardcoded version numbers since it sounds like something that needs manual maintenance as new Python versions come out. Is it possible to just parse the version number from `matrix.python_version_tag` and use that to install the relevant python setup? I think that would be less hassle free to keep running down the line.

Reply:
Actually I see that the following commit changes this already